D/Ethernet/EthernetController.cs

```diff
@@ -720,6 +722,12 @@ private void ReceiverPollCallback(ulong skewNsec, object context)
                         _inputPacket.Enqueue((ushort)(packetStream.ReadByte() << 8));
                     }
 
+                    // add 2 words to ensure that the ethernet task microcode does not ignore the packet
+                    // (the microcode seems to expect a 4 byte "frame check sequence" at the packet end)
+                    _inputPacket.Enqueue((ushort)0);
+                    _inputPacket.Enqueue((ushort)0);

```

Comment:
Are you certain the microcode expects this and that there isn't a different root cause of this issue?  The Ethernet code as it stands works fine under Interlisp-D (tested with the TCP/IP stack) and as far as I know the Ethernet microcode between Interlisp and VP is basically identical.